<ui version="4.0" >
 <author>Pere Constans</author>
 <class>c2bPdfImport</class>
 <widget class="QDialog" name="c2bPdfImport" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>414</height>
   </rect>
  </property>
  <property name="acceptDrops" >
   <bool>true</bool>
  </property>
  <property name="windowTitle" >
   <string>PDFImport - cb2Bib</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QListWidget" name="PDFlist" >
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="Log" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QStatusBar" name="statusBar" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeGripEnabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Abort|QDialogButtonBox::Close|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="Line" name="line1" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Miscellaneous</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="AutomaticQuery" >
        <property name="statusTip" >
         <string>Perform network queries after automatic reference extractions</string>
        </property>
        <property name="text" >
         <string>Perform network queries</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="OpenFiles" >
        <property name="toolTip" >
         <string/>
        </property>
        <property name="statusTip" >
         <string>Open document files</string>
        </property>
        <property name="text" >
         <string>Open document files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Sequence</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="DoNextAfterSaving" >
        <property name="statusTip" >
         <string>Process next after saving</string>
        </property>
        <property name="text" >
         <string>Process next after saving</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="DoAll" >
        <property name="statusTip" >
         <string>Unsupervised processing</string>
        </property>
        <property name="text" >
         <string>Unsupervised processing</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>PDFlist</tabstop>
  <tabstop>Log</tabstop>
  <tabstop>OpenFiles</tabstop>
  <tabstop>DoNextAfterSaving</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="cb2bib.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>c2bPdfImport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>759</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel" >
     <x>822</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
